genai/internal/version.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2022 Google LLC. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package internal
+
+// Version is the current tagged release of the library.
+const Version = "0.172.0"
```

Comment:
It's confusing to have two `internal.Version` meaning different things in the code - is this a necessary structure, or can these version files be placed in packages?

Reply:
I got rid of this one.